docs/rtd-requirements.txt

```diff
@@ -15,7 +15,7 @@ click-plugins==1.1.1
 cligj==0.5.0
 cloudpickle==1.3.0
 commonmark==0.9.1
-cryptography==3.2
+cryptography>=3.3.2
```

Comment:
This file is meant to be fully frozen, so only `==` are allowed. You could try just  bumping the version of that library alone, but it probably won't "just work" as it might have updated dependency constraints in the newer version, or other frozen libraries might have constraints on that library. Since we have no tests for this other than waiting for rtd do use it, we can't tell if this "works" until after merging.

Proper procedure (that ideally would be automated) would be to use something like `pip-tools` to bump versions of all libraries in this file and verify that this works by building a python environment with updated version.

A manual way would be to
1. Build python environment with unchanged file
2. Update library of interest, verify that doesn't break other libs (boto* will probably break), update those libs as needed
3. Dump new frozen set, taking care of things like `pkg-resources==0.0.0` lines
4. Commit the file to git